### DISCUSSION TITLE
How to display choices buttons on two seperate lines in audio-button-response plugin?

### DISCUSSION THREAD

**Discussion by scarlettwwx** [jspsych/jsPsych#1092] [Q&A] [answered]
Let's say we have such piece of code:  
**var capcha = {
          stimulus: "static/mp3s/cat.mp3",
          type: "audio-button-response",
          choices: ['dog','cat','house','children', 'Replay'],
          button_html: '<button class="jspsych-btn">%choice%</button>'}**

If I would like to put **Replay** below the line of _dog, cat, house, children_. How should I modify the code? (Meantime,  I need to get the parameter data.values()[0].button_pressed, i.e. the id of the button pressed, to loop the procedure.)

Thanks for any advice!!

**Comment by becky-gilbert** [comment, marked as answer]
Hi @scarlettwwx! Disclaimer: this might not be the best way, and if someone else has a better suggestion then I hope they'll chime in! But one way to do this is like this:

(1) for the div that contains all of the response buttons (id="jspsych-audio-button-response-btngroup"), set `position: relative` and increase its height so that it can fit two rows of buttons instead of one, and 
(2) for the 'Replay' button's container div (id="jspsych-audio-button-response-button-4"), set `position: absolute` and then adjust `top` and `left` values until it's in the location you want. 

You can do this by adding some CSS rules to your HTML page via `<style>` tags in the `<head>` section:
```html
<style>
      #jspsych-audio-button-response-btngroup {position: relative; height: 100px;}
      #jspsych-audio-button-response-button-4 {position: absolute; top: 50%; left: 40%;}
</style>
```
Or if you only want these changes to be applied to one specific trial, then you can make the CSS changes in the [trial's `on_load` function](https://www.jspsych.org/overview/callbacks/#on_load). In this case you'll have to make the changes via JavaScript:
```javascript
var btn_group_div = document.getElementById("jspsych-audio-button-response-btngroup");
btn_group_div.style.position = "relative";
btn_group_div.style.height = "100px";
var replay_btn_div = document.getElementById("jspsych-audio-button-response-button-4");
replay_btn_div.style.position = "absolute";
replay_btn_div.style.top = "50%";
replay_btn_div.style.left = "40%";
```

CSS pro tip: in the browser's developer tools, you can use the Elements tab to find the elements on the page that you want to modify. Then for each element, you can see the CSS selectors that you can use to identify it (e.g. tag name, ID, class), and you can add/remove/change the CSS and see the result immediately. Once you've figured out what specific changes you want, you can turn them in to CSS rules as above.

**Reply by scarlettwwx** [reply]
Thank you sooo much!!!! It did work! I really appreciate your help!
